dashboard/src/App.js

```diff
@@ -58,6 +59,7 @@ const App = () => {
           <Route path={APP_ROUTES.AUTH} element={<AuthComponent />} />
           <Route path={APP_ROUTES.AUTH_LOGIN} element={<AuthComponent />} />
           <Route path={APP_ROUTES.AUTH_SIGNUP} element={<AuthComponent />} />
+          <Route path="/toc/*" element={<TableOfContent />} />
```

Comment:
@dbutenhof, you and I concur that it would be better to use a named parameter, which I think is an option, but I'm deferring to @shubham-html-css-js on that (once he wakes up 😉).  However, I believe that these are Dashboard routes, not Server API URIs, so there is only one parameter -- the `dataset` in the path is implicit.

Reply:
Yes, they're dashboard routes. In Shubham's demo, the location bar showed `/toc/<long-hex-string>` which represents the dataset resource ID. I pointed out at one point that ideally we want to represent the relative path being displayed as well, so that if you're looking at `/toc/<id>/1-default/directory` you can copy the location URL, email it to me, and I can load it to reproduce exactly the same view. Therefore, we really want this routing to represent both parameters. This isn't a capture of the underlying API URIs, but certainly there's a close correlation.

It may not be necessary to use the named parameters to represent this in React, but that seems like the most straightforward (and easiest to read) implementation.

Reply:
Good point about making the URL copiable, although Shubham did mention in the PR description that that's a "future".  But, either way, I think using named parameters would be better; whether it's one parameter or two in _this_ PR is mostly what I was concerned about (I'm thinking it should be only one -- `dataset`:  in terms of the URL, I had it backwards -- it's the other one which is implicit 🥴).